src/Orleans.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
```

Comment:
what I mean is this diff is not needed for your change, normally we don't include diff which is not necessary,  for reasons like avoiding merge conflicts